docs/examples/affinity/cookie/README.md

```diff
@@ -11,6 +11,8 @@ Session stickiness is achieved through 3 annotations on the Ingress, as shown in
 |nginx.ingress.kubernetes.io/affinity|Sets the affinity type|string (in NGINX only ``cookie`` is possible|
 |nginx.ingress.kubernetes.io/session-cookie-name|Name of the cookie that will be used|string (default to INGRESSCOOKIE)|
 |nginx.ingress.kubernetes.io/session-cookie-hash|Type of hash that will be used in cookie value|sha1/md5/index|
+|nginx.ingress.kubernetes.io/session-cookie-expires|The value is a date as UNIX timestamp that the cookie will expire on, it corresponds to cookie Expires directive|number of seconds|
```

Comment:
This is not accurate now that we made this annotation to accept number of seconds as well (it gets appended to current time to generate date for Expires attribute)